CHANGES.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## 2021-12-14
+
+- Run `node lib/data_migrations/2021-12-14-empty-blocks.js` to reindex all historical empty blocks.
+In addition, reload the following stored procedures:
+
+- `etc/schema/functions/eth_getBlockByNumber.sql`
```

Comment:
can we stick to one migration command ? I think the goose command is more than enough.

Reply:
We can, but we need to make some adjustments to goose migrations to ensure they are working properly.

Reply:
Let's address this issue with different pull request.